pkg/asset/machines/master.go

```diff
@@ -114,9 +138,110 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 	default:
 		return fmt.Errorf("invalid Platform")
 	}
+
+	machines := []interface{}{}
+	if len(m.Machines) == 0 {
```

Comment:
You don't even need to do this check. You can add from both `Machines` and `MachineDeprecated`.

Reply:
> You can add from both `Machines` and `MachineDeprecated`.

Hopefully `MachinesDeprecated` doesn't survive the week, so it's probably not worth quibbling over ;).  But I've taken the approach I have so I can just kill lines and drop tabs for most of the removal (just OpenStack to go).  Is that convincing enough, or did you want me to make this adjutment to satisfy you in the short term?

Reply:
I am fine with making no changes here. It was just a suggestion.